topo/node/ixia/ixia.go

```diff
@@ -83,6 +92,39 @@ func (n *Node) Create(ctx context.Context) error {
 	return nil
 }
 
+func (n *Node) Status(ctx context.Context) (corev1.PodPhase, error) {
+	ixiaNode := Ixia{}
+	var phase corev1.PodPhase
+	err := error(nil)
+	pod, _ := n.KubeClient.CoreV1().Pods(n.Namespace).Get(ctx, n.Name(), metav1.GetOptions{})
+	if pod != nil && pod.Status.Phase == "Running" {
+		return pod.Status.Phase, nil
+	}
+
+	if pod != nil {
+		phase = pod.Status.Phase
+	} else {
```

Comment:
A simpler way to write this is

phase := corev1.PodUnknown
if pod != nil {
    phase = pod.Status.Phase
}

And then you can move the if running check after this.

if pod.Status.Phase == "Running" { ... }

Is there a constant or enum for the "Running" status. Its better than using a raw string here